src/sonic-config-engine/minigraph.py

```diff
@@ -441,7 +443,11 @@ def parse_meta(meta, hname):
                     erspan_dst = value_group
                 elif name == "DeploymentId":
                     deployment_id = value
-    return syslog_servers, dhcp_servers, ntp_servers, tacacs_servers, mgmt_routes, erspan_dst, deployment_id
+                elif name == "KubernetesMaster":
+                    k8s_master = value
+                elif name == "KubernetesInsecure":
+                    k8s_insecure = value
```

Comment:
can you combine them into one dict?

Reply:
It is combined into one in config_db. But if you are talking about it in minigraph, it is added as property, hence read it as above.